opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchDataType.java

```diff
@@ -32,7 +32,7 @@ public enum MappingType {
     Text("text", ExprCoreType.UNKNOWN),
     Keyword("keyword", ExprCoreType.STRING),
     Ip("ip", ExprCoreType.UNKNOWN),
-    GeoPoint("geo_point", ExprCoreType.UNKNOWN),
+    GeoPoint("geo_point", ExprCoreType.GEO_POINT),
```

Comment:
Why not `STRUCT`? Probably, you can even keep `UNKNOWN` until you add functions to `:core` for geo point. Please, test this and revert if it works.